### PR TITLE
QOL-8595 move shared functions to ckanext-qgov extension

### PR DESCRIPTION
**Pull request by ThrawnCA**
This extension should be specific to Open Data.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Retrieve screenshots
         if: failure()
         run: .circleci/process-artifacts.sh
-        timeout-minutes: 3
+        timeout-minutes: 1
```
Comment (duttonw) [inline .github/workflows/test.yml:32]: why reduce time?

File: test/features/login_redirect.feature [hunk 2]
```diff
@@ -2,19 +2,15 @@
...
 
     @dashboard_login
-    Scenario: As an unauthenticated user, when I visit the /dashboard URL I see the login page
+    Scenario: As an unauthenticated user, when I visit the dashboard URL I see the login page
         Given "Unauthenticated" as the persona
         When I visit "/dashboard"
         Then I should see an element with xpath "//h1[contains(string(), 'Login')]"
-
-    @dashboard_login
-    Scenario: As an unauthenticated user, when I visit the /dashboard/ URL I see the login page
```
Comment (duttonw) [inline test/features/login_redirect.feature:11]: given when then,
should not have another when then, re-introduce the scenario again.

**Review by duttonw — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (ThrawnCA) [reply .github/workflows/test.yml:32]: Just matching another repo. This step should never take more than a few seconds.

File: test/features/login_redirect.feature (hunk 2 above)
Comment (ThrawnCA) [reply test/features/login_redirect.feature:11]: At a high level, though, it's the same scenario. The only difference is the trailing slash.

Could make it a parameterised scenario, I suppose?

**Review by duttonw — APPROVED** [review]

**Review by duttonw — APPROVED** [review]

**Review by duttonw — APPROVED** [review]

**Pull request merged**